docs-source/content/faq/oci-fss-pv.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Using OCI File Storage (FSS) for Persistent Volumes"
+date: 2020-02-12T12:12:12-05:00
+draft: false
```

Comment:
Not a show-stopper, but the weight entry is still missing.  The FAQ files in the 'develop' branch look like they all have 'weight:' entries.

Reply:
fixed